lib/display.rb

```diff
@@ -1,13 +1,11 @@
-# frozen_string_literal: true
-
 class Display
-  def present
+  def present(board_values)
     <<~BOARD
-       1 | 2 | 3
+       #{board_values[0]} | #{board_values[1]} | #{board_values[2]}
```

Comment:
Display knows the data structure of the board.  Should display know that?  Suppose you change the data structure of the board?  Then display will fail.  How can you change this design so display does not care about the data structure of board?

Reply:
➕